frontend/src/assets/icons/anchorIcon.svg

```diff
@@ -0,0 +1,23 @@
+<svg width="40px" height="40px" viewBox="0 0 60 60" xmlns="http://www.w3.org/2000/svg">
+ <style>.stb0{stroke:#fff} .stf0{fill:#fff}</style>
+ <g>
+  <title>Layer 1</title>
```

Comment:
This is showing up as a alt-text when hovering over the anchor. I'd remove or change it to `page-anchor`. Better yet, to support accessibility, we should have the alt-text that have the name of the anchored section.